engineapi/alembic/versions/71e888082a6d_leaderboard_metadata.py

```diff
@@ -0,0 +1,53 @@
+"""leaderboard metadata
+
+Revision ID: 71e888082a6d
+Revises: 040f2dfde5a5
+Create Date: 2023-11-15 13:21:16.108399
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "71e888082a6d"
+down_revision = "040f2dfde5a5"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "leaderboards",
+        sa.Column(
+            "blockchain_ids",
+            sa.ARRAY(sa.Integer()),
+            nullable=False,
+            server_default="{}",
+        ),
+    )
+    op.add_column(
+        "leaderboards",
+        sa.Column(
+            "wallet_connect", sa.Boolean(), nullable=False, server_default="false"
+        ),
+    )
+    op.add_column(
+        "leaderboards",
+        sa.Column(
+            "columns_names",
+            postgresql.JSONB(astext_type=sa.Text()),
+            nullable=True,
+            server_default="{}",
```

Comment:
I think better to remove here `server_default` because in function `create_leaderboard` you pass None values here and some of columns will have NUL and some of columns empty dictionary.

Reply:
Also it will reduce checks for both None and {} dictionary